### DISCUSSION TITLE
Old image often showing in "Save Image"

### DISCUSSION THREAD

**Discussion by 42degrees** [comfyanonymous/ComfyUI#1644] [General]
The Save Image node often shows an old image, sometimes an old deleted image.  The folder always contains the newly generated image.  It seems to be happening more often the more images I have in my output folder (currently 282).  If I have it generating a batch of four, then sometimes some of the four are new images, some are old images.  

Is this a known problem?

**Comment by tone-milazzo** [comment]
I'm having the same issue. Firefox 115.3.0esr on Debian 12
